conf.d/50gitlab

```diff
@@ -3,15 +3,13 @@
 source /usr/local/src/gitlab.conf
 [ "$FAB_HTTP_PROXY" ] && export HTTP_PROXY=$FAB_HTTP_PROXY
 
-
-# download and checkout specified version
 URL="https://github.com/gitlabhq/gitlabhq.git"
-
-exec_git "git clone -b $VERSION_GITLAB --single-branch $URL $GIT_HOME/gitlab"
-unset HTTP_PROXY
+exec_git "git clone --single-branch --depth=1 $URL $GIT_HOME/gitlab" 
```

Comment:
FWIW `--depth` implies `--single-branch` (so will only clone a single branch unless `--no-single-branch` is also passed). No biggie, just thought I'd mention...